test/kernel-headers/run-test.sh

```diff
@@ -8,27 +8,42 @@ export EBPF_NET_SRC_ROOT="${EBPF_NET_SRC_ROOT:-$(git rev-parse --show-toplevel)}
 source "${EBPF_NET_SRC_ROOT}/dev/script/bash-error-lib.sh"
 set -x
 
+echo $DOCKER_HUB_PATH
+
 function print {
   echo "===================================== $1 ====================================="
 }
 
-# Takes 2 to 3 arguments: distro, version, and optional kernel version
+# Takes 2 to 4 arguments: distro, version, and optional kernel version, tag
 distro=$1
 version=$2
-kernel_version=$3
-
+if [ $# -eq 3 ];
+then
+  kernel_version=$3
+elif [ $# -eq 4 ] || [ $# -eq 5 ];
+then
+  tag=$4
+fi    
+  
 name=${distro}-${version}
 [ "$kernel_version" != "" ] && name=${name}-${kernel_version}
 print "Testing ${name}"
 ${EBPF_NET_SRC_ROOT}/test/kernel-headers/bootstrap.sh ${distro} ${version} ${kernel_version}
 
 cd ${name}
 print "running 0-setup.sh"
-./0-setup.sh
+./0-setup.sh ${tag}
+
+## this script should only be run for SE linux. Currently only centos-7 is an SE. 
```

Comment:
2-apply-selinux-policy.sh was intended to be run on all distros, and should be safe to run whether the distro is SELinux or not.  Was it causing problems?

Reply:
It was throwing some errors on few of the occasions for Ubuntu-jammy. Do you recommend this to revert?

Reply:
done